pkg/api/v1alpha2/api.pb.gw.go

```diff
@@ -18,8 +18,8 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/grpclog"
 	"google.golang.org/grpc/status"
+	"k8s.io/klog"
```

Comment:
api.pb.gw.go files are generated based on api.proto, I don't think we should modify the files manually

Reply:
DOne